src/main/java/org/commcare/formplayer/application/FormController.java

```diff
@@ -151,9 +154,11 @@ public FormEntryResponseBean answerQuestion(@RequestBody AnswerQuestionRequestBe
     public SubmitResponseBean submitForm(@RequestBody SubmitRequestBean submitRequestBean,
                                          @CookieValue(name=Constants.POSTGRES_DJANGO_SESSION_ID, required=false) String authToken, HttpServletRequest request) throws Exception {
         SerializableFormSession serializableFormSession = formSessionRepo.findOneWrapped(submitRequestBean.getSessionId());
-        FormSession formEntrySession = new FormSession(serializableFormSession, restoreFactory, formSendCalloutHandler, storageFactory, raven);
+        FormSession formEntrySession = new FormSession(serializableFormSession, restoreFactory, formSendCalloutHandler, storageFactory);
         SubmitResponseBean submitResponseBean;
-
+        if (raven != null) {
```

Comment:
It should not be possible for raven to be null, right?

Reply:
Yeah it would signal an issue with Autowired, in which case we would probably want to know about it so makes sense to not have the null check.